src/e3/anod/sandbox/__init__.py

```diff
@@ -171,8 +171,5 @@ def as_requirement(self):  # type: ignore
                     "console_scripts", "sandbox_scripts"
                 )
             with open(target, "wb") as f:
-                if isinstance(script_content, str):
-                    f.write(script_content.encode("utf-8"))
-                else:
-                    f.write(script_content)
+                f.write(script_content.encode("utf-8"))
```

Comment:
why this change? This breaks completely our testsuite on Windows

`C:\projects\e3-core\.tox\py-ci-cov-codecov\lib\site-packages\setuptools\command\easy_install.py:2097: EasyInstallDeprecationWarning: Use get_args
  warnings.warn("Use get_args", EasyInstallDeprecationWarning)
C:\projects\e3-core\.tox\py-ci-cov-codecov\lib\site-packages\setuptools\command\easy_install.py:2099: EasyInstallDeprecationWarning: Use get_header
  header = cls.get_script_header("", executable, wininst)
Traceback (most recent call last):
  File "C:\Python37\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "C:\Python37\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\projects\e3-core\.tox\py-ci-cov-codecov\Scripts\e3-sandbox.EXE\__main__.py", line 7, in <module>
  File "C:\projects\e3-core\.tox\py-ci-cov-codecov\lib\site-packages\e3\anod\sandbox\main.py", line 75, in main
    ext[args.action].obj.run(args)
  File "C:\projects\e3-core\.tox\py-ci-cov-codecov\lib\site-packages\e3\anod\sandbox\action.py", line 95, in run
    sandbox.write_scripts()
  File "C:\projects\e3-core\.tox\py-ci-cov-codecov\lib\site-packages\e3\anod\sandbox\__init__.py", line 174, in write_scripts
    f.write(script_content.encode("utf-8"))
AttributeError: 'bytes' object has no attribute 'encode'
---------------------------- Captured stderr call -----------------------------
`